app/src/main/java/com/example/vmedvediev/kotlintimer/AlarmManager.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.vmedvediev.kotlintimer
+
+import android.app.AlarmManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import com.example.vmedvediev.kotlintimer.util.PrefUtil
+
+object AlarmManager {
+
+    fun setAlarm(context: Context, nowSeconds: Long, secondsRemaining: Long): Long{
+        val wakeUpTime = (nowSeconds + secondsRemaining) * 1000
```

Comment:
What is `1000` ? Such unknown numbers are called "Magic numbers" - and it is "code smell".